README.md

```diff
@@ -1,2 +1,49 @@
-# moodster-app
+# Moodster
 Moodster Application Project
+
+# Setting up the Development Environment
+
+Go to https://facebook.github.io/react-native/docs/getting-started.html, and select the **React Native CLI Quickstart** tab as this project doesn't use Expo. **Follow these instructions to setup your development environment.**
+
+### Windows/Linux
+
+You will need Node, the React Native command line interface, Python2, a JDK, and Android Studio.
+
+### macOS
+ 
+You will need Node, Watchman, the React Native command line interface, and Xcode.
+
+*Note: A Mac is required to build projects with native code for iOS.*
+
+
+# Running the App
+Once you've installed the relevant dependancies and have either a phyiscal/virtual device connected. Run the following commands
+```
+cd moodster-app
+npm install
+```
+
+### Android
+```
+react-native run-android
+```
+
+### iOS
+```
+react-native run-ios
+```
+
+For the app to connect to the local API correctly, follow the installation steps for https://github.com/JBonser/moodster-api.
+If you are running the app on a phyiscal device, the device will need to be on the same network as the host machine and you will need to follow these steps:
+
+### Android
+
+1) Connect Android mobile device with USB cable to laptop
```

Comment:
I think there is actually a better way to do this that using adb reverse.
Flask applications allow you to specify a URI of 0.0.0.0:5000 rather than 127.0.0.1. The difference being that with the 0 version it makes that port available to any device on the same network.

This approach works for now but it is something we should look into as to keep the setup complexity down.

Reply:
Yeah agree, this problem only occurs with a physical android device and is technically nothing to do with this app. Don't mind removing this section completely from the readme tbh. If we can change the flask app so it operates in the way you suggested that would be much better.